samples/matter/common/src/Kconfig

```diff
@@ -39,3 +39,19 @@ config NCS_SAMPLE_MATTER_FACTORY_RESET_ON_KEY_MIGRATION_FAILURE
 	help
 	  Allow device to perform factory reset if the operational key for Fabric has not been migrated
 	  properly to PSA ITS storage.
+
+config NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS
```

Comment:
It's fair to not enable it by default, but I think we should enable it for door lock in its prj.conf/Kconfig, as we are going to test it during TE.

Reply:
OK, enabling it by default.

Reply:
I mean enabling it for **door lock in its** prj.conf/Kconfig not for all samples :smile: Nvm, we can merge it as is, because it's only for test event purposes.